src/azure-cli-core/HISTORY.rst

```diff
@@ -2,10 +2,9 @@
 
 Release History
 ===============
-
 2.0.50
 ++++++
-* Fix issue where update commands using `--remove` and `--ids` fail after first update is applied to first resource in ids list.
+* auth: support service principal sn+issuer auth
```

Comment:
Was the previous history change unnecessary to include?

Reply:
The deleted history entry was part of 2.0.49.